src/libsync/platform_mac.mm

```diff
@@ -39,20 +44,115 @@ - (BOOL)applicationShouldHandleReopen:(NSApplication *)sender hasVisibleWindows:
     if (auto *app = QApplication::instance()) {
         QMetaObject::invokeMethod(app, "showSettingsWindow", Qt::QueuedConnection);
     } else {
-        qDebug() << "Failed to call showSettingsWindow slot";
+        qCDebug(OCC::lcPlatform) << "Failed to call showSettingsWindow slot";
     }
     return YES;
 }
 
 @end
 
+namespace {
+
+class PowerNotificationsListener
+{
```

Comment:
Should we mention the source of inspiration?

Reply:
done